skyvern/forge/sdk/db/client.py

```
@@ -3786,6 +3786,31 @@ async def create_task_run(
+        title: str | None = None,
+        url: str | None = None,
+        url_hash: str | None = None,
+    ) -> None:
+        async with self.Session() as session:
+            task_run = (
+                await session.scalars(
+                    select(TaskRunModel).filter_by(run_id=run_id).filter_by(organization_id=organization_id)
+                )
+            ).first()
+            if not task_run:
+                raise NotFoundError(f"TaskRun {run_id} not found")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)